clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -181,18 +180,19 @@ public HubConnectionState getConnectionState() {
      * Starts a connection to the server.
      * @return A completable future that completes when the connection has been established.
```

Comment:
Update these return docs to reference Completable instead of CompletableFuture

@dend If we want to link to external Javadocs (for third-party libraries) can we do that in this way? The `Completable` type here comes from RxJava. Can we make something like `{@link Completable}` generate a link to reactivex.io/RxJava/javadoc/io/reactivex/Completable.html when the docs are built?